src/test/scala/org/zalando/spearheads/innkeeper/services/RoutesServiceSpec.scala

```diff
@@ -325,7 +325,9 @@ class RoutesServiceSpec extends FunSpec with Matchers with MockFactory with Scal
     ownedByTeam = ownedByTeam,
     createdBy = createdBy,
     createdAt = createdAt,
-    updatedAt = updatedAt)
+    updatedAt = updatedAt,
+    hasStar = false
```

Comment:
do we test that when a `has_star` is present in the path, that path will and up with a * in the DB?